cpp/mpi_comms/src/cuML_comms_mpi_impl.cpp

```diff
@@ -175,14 +175,14 @@ namespace {
 #endif
 }
 
-void initialize_mpi_comms(cumlHandle& handle, MPI_Comm comm)
+void initialize_comms(cumlHandle& handle, MPI_Comm comm)
```

Comment:
Why replace the existing `initialize_mpi_comms` and not just provide an additional `initialize_ucxnccl_comms` implementation in a new file? I think there is value in keeping both version side by side.